refs/biblio.json

```diff
@@ -2708,12 +2708,27 @@
             "Renato Iannella",
             "Serena Villata"
         ],
-        "href": "http://w3c.github.io/poe/model/WD/",
+        "href": "http://www.w3.org/TR/odrl-model/",
         "publisher": "W3C",
-        "status": "W3C Editors Draft",
-        "rawDate": "2016-06-20",
+        "status": "W3C First Public Working Draft",
+        "rawDate": "2016-07-21",
         "title": "ODRL Information Model"
     },
+    "odrl-w3c-vocab": {
```

Comment:
Legacy W3C entries which are in biblio should just be aliased to their automatically generated entry in the W3C file.